Detectors/GlobalTrackingWorkflow/src/PrimaryVertexingSpec.cxx

```diff
@@ -115,15 +115,18 @@ void PrimaryVertexingSpec::run(ProcessingContext& pc)
     if (!_origID.includesDet(DetID::ITS)) {
       return true; // just in case this selection was not done on RecoContainer filling level
     }
-    if constexpr (isITSTrack<decltype(_tr)>()) {
+    if constexpr (isITSTrack<decltype(_tr)>() || isMFTTrack<decltype(_tr)>()) {
```

Comment:
this is a track pool creation for the vertex finding, where I require all tracks to be contributed by the ITS. So, the MFT tracks will no pass anyway. For the time-matching you should add MFT selection not here but in https://github.com/AliceO2Group/AliceO2/blob/0ded4494eac12c34fec80bed85625b661a82e921/Detectors/Vertexing/src/VertexTrackMatcher.cxx#L161